model/flow/incorporated_result.go

```diff
@@ -18,22 +16,12 @@ type IncorporatedResult struct {
 	// Result is the ExecutionResult contained in the ExecutionReceipt that was
 	// incorporated in the payload of IncorporatedBlockID.
 	Result *ExecutionResult
-
-	// chunkApprovals is a placeholder for attestation signatures
-	// collected for each chunk. It gets populated by the consensus matching
-	// engine when approvals are matched to execution results.
-	// This field is not exported (name doesn't start with a capital letter), so
-	// it is not used in calculating the ID and Checksum of the Incorporated
-	// Result (RLP encoding ignores private fields).
-	chunkApprovals     map[uint64]*SignatureCollector
-	chunkApprovalsLock sync.Mutex
```

Comment:
Thank you for removing this. There is only one place where `SignatureCollector` (implemented in this file) is used: the sealing logic. Would suggest to move the  `SignatureCollector` from package flow to package `approvals`.